ui/raidboss/data/00-misc/test.ts

```diff
@@ -365,6 +365,29 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'Test Countdown',
+      type: 'Countdown',
+      netRegex: { result: '00' },
+      infoText: (_data, matches, output) =>
+        output.countdown!({ player: matches.name, seconds: matches.countdownTime }),
+      outputStrings: {
+        countdown: {
+          en: '${player} started ${seconds}s countdown',
+        },
+      },
+    },
```

Comment:
Maybe add an additional test trigger here for `Test Countdown Failed`, where `result` !== `00`, showing the result code?